### PR TITLE
gdub: new port

### PR DESCRIPTION
**Pull request by breun**
###### Description

Added a new port for gdub.

###### Tested on
macOS 10.12.5
Xcode 8.3

###### Verification 
Have you
- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by raimue** [comment]
Should this depend on gradle? Maybe `notes {}` could hint that you also need to install gradle manually?

**Comment by breun** [comment]
Gdub can be used without Gradle installed system-wide. The Gdub site says: "You will probably want to install Gradle first. While this is not technically necessary if all your projects are using a Gradle Wrapper, it is a good idea to have the latest version of gradle available system-wide because some handy Gradle features are available outside the context of an existing project." So installing Gradle system-wide is optional.

File: devel/gdub/Portfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+homepage            http://www.gdub.rocks/
+
+checksums           rmd160  fa612e31fd2f32e429225338ff5361184b82b318 \
+                    sha256  76d9a81ba295cf8a4024bde845cd8e36bb01e3465ed09221fb69d7627150afbc
+
+use_configure       no
+build {}
+
+pre-destroot {
+    delete ${worksrcpath}/install
```
Comment (raimue) [inline devel/gdub/Portfile]: Is there a reason to delete this? It does not look like this path would even be used in the destroot phase at all.

**Comment by ryandesign** [comment]
Please add the standard modeline.

File: devel/gdub/Portfile (hunk 1 above)
Comment (breun) [inline devel/gdub/Portfile]: I guess it would work fine without this, so I'll remove this.

**Comment by breun** [comment]
https://guide.macports.org/chunked/development.creating-portfile.html says the following about the modeline: "Its use is optional and up to the port maintainer." But I'm fine with adding it.

**Comment by breun** [comment]
Is it possible to check whether `gradle` is available in the `$PATH` and if not show `notes`? If so, what is the syntax for that? If all of your projects use Gradle Wrapper it is not really necessary to have a system-wide installation of Gradle, so I didn't want to make it a real dependency.

**Comment by raimue** [comment]
In my opinion `port notes` should not be conditional. The user's environment is not available to a Portfile and we do not even know it at all when running with sudo. I assume users know how to work with this port.

**Pull request closed without merging**